nrt/data/DynaFlow/GeneratorPVTfo_rpcl_rpcl2/GeneratorPVTfo_rpcl_rpcl2.crv

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The name of the folder could be something like GeneratorPVTfoRpclTimeline, what do you think ?